package.json

```diff
@@ -20,23 +20,33 @@
     "three": ">=0.118.0"
   },
   "scripts": {
-    "build": "gulp",
-    "test": "gulp test"
+    "build": "webpack --mode=production --config-name=threestrap",
+    "dev": "webpack --mode=development --watch --config-name=threestrap",
+    "prepack": "npm run build",
+    "lint": "eslint src/**/*.js",
```

Comment:
**Note:** Not currently linting tests.

I expect to add test linting (and overall formatting) in a subsequent PR. Tests currently contain a bunch of linting errors, and I'd rather not do more in this PR.